CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Changelog
 
+# Version 1.0.0
+
+## Address
+
+Safe Locking Contract Address: [0x0a7CB434f96f65972D46A5c1A64a9654dC9959b2](https://etherscan.io/address/0x0a7CB434f96f65972D46A5c1A64a9654dC9959b2)
```

Comment:
In this case I think it kind of makes sense. The CHANGELOG follows historical versions, while the README is the current version, and is the landing page for what external devs would see. So when they come to page they would immediately see where the contract is.

If we really want to be strict against duplication of this information, then the README should link to the CHANGELOG so that external developers that come to this repository know where to look for the contract address.

Reply:
As this is a mainnet production-ready address, I also think it is fine to keep both, considering if there is a future version, then `README.md` will be updated, though `CHANGELOG.md` won't be edited, though updated with a new address.